start.py

```diff
@@ -883,7 +883,7 @@ def set_up_logging_file():
             [
                 "sed",
                 "--unbuffered",
-                "s|^[0-9\-]\+\s[0-9:\.]\+\s||",
+                r"s|^[0-9\-]\+\s[0-9:\.]\+\s||",
```

Comment:
As above, normal string literal to a raw string is a functional change. What is the rationale for this, and why is it grouped with the library upgrades commit?

Reply:
no longer in mr.